src/commands.ts

```diff
@@ -175,6 +175,28 @@ export function registerCommands(context: vscode.ExtensionContext, prManager: Pu
 		}
 
 		vscode.commands.executeCommand('vscode.diff', parentURI, headURI, fileName, opts);
+		if (isPartial) {
```

Comment:
I think this should still be above the `vscode.diff` command

Reply:
Sounds good. I changed 'await' to '.then()' to make sure the diff still comes up when the user hasn't selected an option in the information message yet.